### ISSUE TITLE
Please make vs shaggy and vs shaggy v2, with 6 & 9 key arepe with 6 & 9 button please:D

### ISSUE THREAD

**Issue by Neturry** [luckydog7/Funkin-android#1048]

**Comment by hrulol** [comment]
How will nine keys exist when controls are only 4 notes?

**Comment by nsebheebsjbdn** [comment]
> How will nine keys exist when controls are only 4 notes?

WTF

**Comment by TankPain** [comment]
Someone us making it

**Comment by bloodheroC** [comment]
Who

**Comment by netonero** [comment]
modchart.lua is impossible to be compatible, especiallh when its hardcoded
Since most phones would just go ballistic and crash

**Comment by bloodheroC** [comment]
Then if you want shaggy mod then buy a pc

**Comment by hakerSALAD** [comment]
> Then if you want shaggy mod then buy a pc

Shut up

**Comment by FharelYusinardi** [comment]
> Then if you want shaggy mod then buy a pc

You think everyone can afford to buy a pc

**Comment by bloodheroC** [comment]
Heh they said they need shaggy mod its so hard to code so you need a pc to code too

**Comment by TankPain** [comment]
Someone called peppy is making it chill guys

**Comment by TankPain** [comment]
![Uploading Screenshot_20210530_152817.jpg…]()

**Comment by bloodheroC** [comment]
> > Then if you want shaggy mod then buy a pc
> 
> Shut up

Shut up too they want shaggy then they need a pc to code it

**Comment by FharelYusinardi** [comment]
> Heh they said they need shaggy mod its so hard to code so you need a pc to code too

who said that?

**Comment by bloodheroC** [comment]
> > Heh they said they need shaggy mod its so hard to code so you need a pc to code too
> 
> who said that?

The arrows and animation and dialogue and so many things

**Comment by Qusaithedemon** [comment]
> > > Heh they said they need shaggy mod its so hard to code so you need a pc to code too
> > 
> > 
> > who said that?
> 
> The arrows and animation and dialogue and so many things

yeah actually thats kinda true :/ thats how they make mods with dialogs on mobiles and cutscenes too if they make em then its coding and you can make a costume week from coding so its true you need a pc to code a good mod with cutscenes and dialogs

**Comment by bloodheroC** [comment]
Yea but its super hard

**Comment by bloodheroC** [comment]
So the ez way is just the pc

**Comment by FnfWeek7** [comment]
#1064 it's already made. -_-

**Comment by Elmorado** [comment]
@FnfWeek7 XD

**Comment by hakerSALAD** [comment]
https://www.mediafire.com/file/9k0rht0sft4uid4/Funkin-debug-sh%2Aggy-2.apk/file

**Comment by FnfWeek7** [comment]
> @FnfWeek7 XD

Thanks dude can u make restored next when luckydog make
